vpr/src/device/rr_graph_obj.cpp

```diff
@@ -449,6 +449,7 @@ void RRGraph::print_node(const RRNodeId& node) const {
     VTR_LOG("Node ptc: %d\n", node_ptc_num(node));
     VTR_LOG("Node num in_edges: %d\n", node_in_edges(node).size());
     VTR_LOG("Node num out_edges: %d\n", node_out_edges(node).size());
+    VTR_LOG("Node segment id: %d\n", node_segment(node));
```

Comment:
Not every node has a valid segment id. I suggest to output segment id only when the node is CHANX or CHANY

Reply:
Ok, I've fixed that.